lib/virtmanager.pm

```diff
@@ -15,9 +15,11 @@ sub launch_virtmanager() {
     type_string "virt-manager", 50;
     send_key "ret";
     wait_idle;
-    type_password;
-    send_key "ret";
-    wait_idle;
+    if (check_var("FLAVOR", "Server-DVD")) {
```

Comment:
but what will happen on TW? I think it scales better if you replace this hard coded flavor things with check_screens